### PR TITLE
test(MM-54404): Add e2e tests for gif picker

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
Add manual test steps for Gif picker

#### Is the feature in stable branch or under review?
- [X] Feature in stable branch (master, main)?
- [ ] Feature still under review? If yes, list all related pull requests for reference.

#### Test server
n/a

#### Test client

- [X] Browser
- [ ] Mobile App (version, OS)
- [X] Desktop App (version, OS)

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
⚠️ 
File names are pending please add Test keys

File: data/test-cases/channels/gif-picker/insert-key1.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+# (Required) Ensure all values are filled up
+name: "Change Test Title"
```
Comment (yasserfaraazkhan) [inline data/test-cases/channels/gif-picker/insert-key1.md:3]: hi @M-ZubairAhmed maybe we can add a title to these

**Review by yasserfaraazkhan — CHANGES_REQUESTED** [review]

File: data/test-cases/channels/gif-picker/insert-key1.md (hunk 1 above)
Comment (M-ZubairAhmed) [reply data/test-cases/channels/gif-picker/insert-key1.md]: Done ✅

**Review by saturninoabril — APPROVED** [review]
Awesome, thanks @M-ZubairAhmed!

**Review by yasserfaraazkhan — APPROVED** [review]
🎉

**Comment by yasserfaraazkhan** [comment]
> ⚠️ File names are pending. Please add Test keys
> cc @yasserfaraazkhan

@M-ZubairAhmed 
MM-T5445 : "Should search, select and post correct Gif when Gif picker is opened from center textbox"

MM-T5446 : "Should search, select and post correct Gif when Gif picker is opened from RHS textbox"

**Pull request merged**